src/main/scala/chisel3/simulator/scalatest/HasCliOptions.scala

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3.simulator.scalatest
+
+import chisel3.testing.scalatest.HasConfigMap
+import firrtl.options.StageUtils.dramaticMessage
+import org.scalatest.TestSuite
+import scala.collection.mutable.HashMap
```

Comment:
```suggestion
import scala.collection.mutable
```
Style nit but never import specific things from `scala.collection.mutable`, instead use `mutable.HashMap` below.

Reply:
Fixed in https://github.com/chipsalliance/chisel/pull/4777/commits/fdf82cf80c0dbdd73acce2bd9e6eb697aca2c069.